docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,189 @@
-# How To Use
+# AdnSkin
 
-Skin is a Maya deformer for fast, robust and easy-to-configure skin simulation for digital assets. Thanks to the combination of internal and external constraints, the deformer can produce dynamics that allow the skin mesh to realistically react to the deformations of the internal tissues (e.g. muscles, fascia) over time.
+AdnSkin is a Maya deformer for fast, robust and easy-to-configure skin simulation for digital assets. Thanks to the combination of internal and external constraints, the deformer can produce dynamics that allow the skin mesh to realistically react to the deformations of the internal tissues (e.g. muscles, fascia) over time.
+
+<!-- DEV NOTE: change paint tool references once tools.md has been merged-->
+
+The influence these constraints have on the simulated mesh can be freely modified by painting them via the [AdonisFX Paint Tool](index.md) or by uniformingly regulating their influence via multipliers in the attribute editor. Besides the maps and multipliers there are many other parameters to regulate the skin's dynamics and behaviour to a wide array of options.
+
+# How to Use
+
+The AdnSkin deformer is of great simplicity so set up and apply to a mesh within a Maya scene. The way this deformer works, a reference mesh (usually animated) is set in the scene, over which the skin mesh (simulated mesh) is applied with the deformer.
 
 ## Requirements
 
-The Skin deformer requires the following inputs to be provided:
+To create an AdnSkin deformer within a Maya scene, the following inputs must be provided:
+
+  - <b class="mesh_color"> Reference Mesh (R)</b>: Mesh to drive the simulation skin (e.g. fascia or combined muscles).
+  - <b class="mesh_color"> Skin Mesh (S)</b>: Mesh to be simulated to which the AdnSkin deformer will be applied.
```

Comment:
I think that we must remove "class" or "style" found since styling will come from the web.